modules/storages/app/models/storages/storage.rb

```diff
@@ -53,6 +53,7 @@ class Storage < ApplicationRecord
     self.inheritance_column = :provider_type
 
     store_attribute :provider_fields, :automatically_managed, :boolean
+    store_attribute :provider_fields, :health_notifications_enabled, :boolean, default: true
```

Comment:
I remember problems with this default values. They only work for new records as far as I remember. But I do not see it as a problem in this case. I think it means that existing storages should enable notification explicitly from the UI.

Reply:
Ah, good spot- I think we can base it off `automatically_managed` as we have the AC that it should be on by default for all storages with automatic management.

Thoughts on this instead?

```ruby
store_attribute :provider_fields, :health_notifications_enabled, :boolean

def health_notifications_enabled
  super.presence || automatic_management_enabled?
end
```